packages/dx-react-grid/src/plugins/table-header-row.jsx

```diff
@@ -46,27 +46,26 @@ export class TableHeaderRow extends React.PureComponent {
           {params => (
             <TemplateConnector>
               {({
-                sorting, grouping, columns,
+                sorting, tableColumns, draggingEnabled,
               }, {
                 setColumnSorting, groupByColumn,
                 changeTableColumnWidths, changeDraftTableColumnWidths,
               }) => {
                 const { name: columnName } = params.tableColumn.column;
-                const groupingSupported = grouping !== undefined &&
-                    grouping.length < columns.length - 1;
+                const isNotTheLastVisible = tableColumns
```

Comment:
isTheLastInvisible?

Reply:
Not sure that I understand how to use the find method here.

Reply:
```js
const atLeastOneVisible = tableColumns.findIndex(({ type }) => type === TABLE_DATA_TYPE) !== -1;
```

Reply:
Ok, but I need to check that there are 2 and more columns.

Reply:
Oh, sorry, missed this `length > 1`. Anyway, `isNotTheLastVisible` sounds strange)

Reply:
I've changed it to `atLeastOneDataColumn`